piIOComm.c

```diff
@@ -41,6 +41,7 @@
 #include <linux/kthread.h>
 #include <linux/gpio.h>
 #include <linux/jiffies.h>
+#include <linux/pibridge_comm.h>
```

Comment:
is linux/pibridge_comm.h missing in linux repo ?

Reply:
Indeed, it is. I will add it and push again, thx.